app/clear_local.sh

```
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Get the absolute path to the script's directory, regardless of where it's run from
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+
+# Change to the app directory if we're not already there
+cd "$SCRIPT_DIR"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)